### PR TITLE
Fix (Airdrops): `has_decoder` field

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Changes

- [x] Add `has_decoder` field as false for the airdrops that don't have any supported decoder yet.
- [x] Remove the eigenlayer airdrop.
- [x] Move the airdrop docs to the contribution guide.

### CODE REVIEW

File: tests/airdrop_index_v2_schema.json [hunk 1]
```diff
@@ -19,19 +19,35 @@
...
                         },
                         "url": {
                             "type": "string",
                             "format": "uri"
                         },
-                        "name": {"type": "string"},
-                        "icon": {"type": "string"},
-                        "icon_path": {"type": "string"},
-                        "cutoff_time": {"type": "integer"},
+                        "name": {
```
Comment (LefterisJP) [inline tests/airdrop_index_v2_schema.json:36]: Why this change?

File: airdrops/index_v2.json [hunk 2]
```diff
@@ -155,6 +161,7 @@
             "icon": "starknet.svg",
```
Comment (LefterisJP) [inline airdrops/index_v2.json:161]: The myso finance airdrop seems to not have an entry and afaik it's **not** decoded.

File: tests/airdrop_index_v2_schema.json (hunk 1 above)
Comment (OjusWiZard) [reply tests/airdrop_index_v2_schema.json:36]: is due to auto-formatting it, earlier it was inconsistent

Comment (OjusWiZard) [reply tests/airdrop_index_v2_schema.json:36]: also added the `has_decoder` field in the schema

**Review by LefterisJP — APPROVED** [review]
LGTM
